src/pheval_exomiser/post_process/post_process_results_format.py

```diff
@@ -97,25 +100,55 @@ def _find_relevant_score(self, result_entry) -> float:
         """Return score from Exomiser result entry."""
         return round(result_entry[self.score_name], 4)
 
-    def extract_pheval_variant_requirements(self) -> [PhEvalVariantResult]:
+    def _filter_for_acmg_assignments(
+        self, variant: PhEvalVariantResult, score: float, variant_acmg_assignments: dict
+    ) -> bool:
+        """Filter variants if they meet the PATHOGENIC or LIKELY_PATHOGENIC ACMG classification."""
+        for assignment in variant_acmg_assignments:
+            if variant == PhEvalVariantResult(
+                chromosome=self._find_chromosome(assignment["variantEvaluation"]),
+                start=self._find_start_pos(assignment["variantEvaluation"]),
+                end=self._find_end_pos(assignment["variantEvaluation"]),
+                ref=self._find_ref(assignment["variantEvaluation"]),
+                alt=self._find_alt(assignment["variantEvaluation"]),
+                score=score,
+            ):
+                if (
+                    assignment["acmgClassification"]
```

Comment:
```python
        for assignment in variant_acmg_assignments:
            if variant == PhEvalVariantResult(
                chromosome=self._find_chromosome(assignment["variantEvaluation"]),
                start=self._find_start_pos(assignment["variantEvaluation"]),
                end=self._find_end_pos(assignment["variantEvaluation"]),
                ref=self._find_ref(assignment["variantEvaluation"]),
                alt=self._find_alt(assignment["variantEvaluation"]),
                score=score,
            ):
                if (
                    assignment["acmgClassification"]
                     == PhEvalVariantResultFromExomiserJsonCreator.acmg_pathogenic_label
```
can't you use an `and` instead of a nested `if`? 

```python
        for assignment in variant_acmg_assignments:
            variant_evaluation = assignment["variantEvaluation"]
            acmg_classification = assignment["acmgClassification"]
            if variant == PhEvalVariantResult(
                chromosome=self._find_chromosome(variant_evaluation),
                start=self._find_start_pos(variant_evaluation),
                end=self._find_end_pos(variant_evaluation),
                ref=self._find_ref(variant_evaluation,
                alt=self._find_alt(variant_evaluation),
                score=score
            ) and (acmg_classification == "PATHOGENIC" or acmg_classification == "LIKELY_PATHOGENIC"):
```

Reply:
also see use of local variables rather than repeatedly looking up the same value from a dictionary.